### PR TITLE
TriggerBxMonitor: migrate to Stage 2

### PR DESCRIPTION
**Pull request by fwyzard**
TriggerBxMonitor
- migrate to Stage 2 L1 Trigger
- correct the number of bx to 3564
- add 2D plots, remove plots for unused L1 triggers
- update test

### CODE REVIEW

**Comment by fwyzard** [comment]
please test

**Comment by fwyzard** [comment]
+1

File: DQM/HLTEvF/plugins/TriggerBxMonitor.cc [hunk 1]
```diff
@@ -68,52 +66,48 @@ class TriggerBxMonitor : public DQMEDAnalyzer {
...
 
 
 
 void TriggerBxMonitor::fillDescriptions(edm::ConfigurationDescriptions & descriptions)
 {
   edm::ParameterSetDescription desc;
-  desc.addUntracked<edm::InputTag>( "l1tResults",       edm::InputTag("gtDigis"));
-  desc.addUntracked<edm::InputTag>( "hltResults",       edm::InputTag("TriggerResults"));
-  desc.addUntracked<std::string>(   "dqmPath",          "HLT/TriggerBx" );
+  desc.addUntracked<edm::InputTag>( "l1tResults", edm::InputTag("gtDigis"));
```
Comment (Martin-Grunewald) [inline DQM/HLTEvF/plugins/TriggerBxMonitor.cc]: Should be "gtStage2Digis"

**Comment by fwyzard** [comment]
please test

**Comment by fwyzard** [comment]
replaced by #14349

**Pull request closed without merging**